k8s/kube-base/cron-job-xdmod-ingestor.yaml

```diff
@@ -9,12 +9,27 @@ spec:
       backoffLimit: 1
       template:
         spec:
+          initContainers:
+            - name: download-config
+              image: xdmod-openstack
+              imagePullPolicy: IfNotPresent
+              command:
+                - "/app/xdmod-get-config.sh"
+              volumeMounts:
+                - name: vol-xdmod-conf
+                  mountPath: /etc/xdmod
+                - name: vol-clouds-yaml
+                  mountPath: /etc/openstack
+                - name: vol-xdmod-init
+                  mountPath: /mnt/xdmod_init
+                - name: vol-var-log-xdmod
+                  mountPath: /var/log/xdmod
           containers:
             - name: moc-xdmod
               image: moc-xdmod
               imagePullPolicy: IfNotPresent
               command:
-                - "/usr/bin/xdmod-run-ingestor.sh"
```

Comment:
Since this file is not being used anymore it might be worth deleting it from the repo.

Reply:
Yup eliminated it.  You caught this after I created it, but before I moved it to draft.

Reply:
I haven't quite finished testing this and was planning to remove that file.  which is why I hadn't assigned any people to review.